.eslintrc.js

```diff
@@ -6,24 +6,42 @@ const config = {
         es2020: true,
         node: true,
     },
-    extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+    extends: [
+        'eslint:recommended',
+        'plugin:node/recommended',
+        'plugin:import/errors',
+        'plugin:import/warnings',
+        'plugin:prettier/recommended',
+    ],
     parserOptions: {
         ecmaVersion: 11,
         sourceType: 'module',
     },
     overrides: [
         {
             files: '**/*.ts',
-            extends: 'plugin:@typescript-eslint/recommended',
+            extends: ['plugin:@typescript-eslint/recommended', 'plugin:import/typescript'],
             parser: '@typescript-eslint/parser',
             plugins: ['@typescript-eslint'],
             rules: {
                 'no-unused-vars': ['error', { argsIgnorePattern: '^_', varsIgnorePattern: '^_' }],
                 '@typescript-eslint/no-unused-vars': ['warn', { argsIgnorePattern: '^_', varsIgnorePattern: '^_' }],
+                'node/no-missing-import': [
+                    'error',
+                    {
+                        tryExtensions: ['.js', '.d.ts', '.ts'],
+                    },
+                ],
+                'node/no-unsupported-features/es-syntax': [
+                    'error',
+                    {
+                        ignores: ['modules'],
+                    },
+                ],
             },
         },
         {
-            files: '**/*.test.{ts,js}',
```

Comment:
`.js` files in `test-packages` need to be run as well.
Only testing against `.ts`, can mean the final `.js` files do not actually work.

Reply:
Hmm, `test-packages` do not use jest.

Reply:
Not sure I follow. This is for the linting targets and not the jest execution. All the compiled TS files in the `dist` folder are ignored

Reply:
My confusion. It was a bit too late when I was reading this.